src/dshould/package.d

```diff
@@ -281,15 +281,26 @@ unittest
         ("Test failed: expected Nullable.null, but got 42");
 }
 
-@("nullable equality")
+@("nullable equality with value")
 unittest
 {
     import std.typecons : Nullable;
 
     Nullable!string().should.not.equal("");
 
     Nullable!string().should.equal("").should.throwA!FluentException
-        ("Test failed: expected value == '', but got Nullable!string.null");
+        ("Test failed: expected '', but got Nullable.null");
+}
+
+@("value equality with nullable")
+unittest
+{
+    import std.typecons : Nullable;
+
+    "".should.not.equal(Nullable!string());
+
+    "".should.equal(Nullable!string()).should.throwA!FluentException
```

Comment:
I would prefer line breaks between uses of `should` and "meta" uses of `should`:

    "".should.equal(Nullable!string())
        .should.throwA!FluentException("Test failed: expected Nullable.null, but got ''");

Reply:
done